test/run

```diff
@@ -427,6 +427,7 @@ TEST_SUMMARY=''
 ct_enable_cleanup
 
 CID_FILE_DIR=$(mktemp -d)
+cid_file="$CID_FILE_DIR/$(mktemp -u -p . --suffix .cid)"
```

Comment:
I don't follow, why this? Can you please paste a link to why cid_file should be created?

Reply:
In the `test/run` is the `cid_file` always used for storing the id of created container. Untill now this variable has not been set properly.
Example: https://github.com/sclorg/nginx-container/blob/ac850b9244dff752a472cee429aab88e3ea30fd0/test/run#L80